src/main/java/uk/gov/companieshouse/filinghistory/consumer/service/FormTypeService.java

```diff
@@ -15,4 +19,8 @@ public FormTypeService(List<String> formTypeBlacklist) {
     public boolean isAssociatedFilingBlacklisted(TransactionKindCriteria criteria) {
```

Comment:
Now we are in the 21st Century can we avoid terms like 'blacklist'? Our colleagues in the National Cyber Security Centre would have us use terms like ‘blocklist’ and ‘allowlist’.